evalml/pipelines/utils.py

```diff
@@ -170,9 +170,6 @@ def generate_pipeline_code(element):
         raise ValueError("Element must be a pipeline instance, received {}".format(type(element)))
 
     component_graph_string = ', '.join([com.__class__.__name__ if com.__class__ not in all_components() else "'{}'".format(com.name) for com in element.component_graph])
-    import_strings = [com.__class__.__name__ for com in element.component_graph if com.__class__ in all_components()]
```

Comment:
Removed this because for EvalML components, we don't need to import them in order to reference and use them in the component graph of a pipeline